scripts/benchmark_gimlet_script.sh

```diff
@@ -2,6 +2,21 @@ export GIMLET_CONTROLPLANE_BASE="https://localhost:60002"
 export GIMLET_API_KEY=""
 export GIMLET_DEVICE_ID=""
 
+# Script to warm-up the gimlet controlplane.
+curl -X GET \
+  "$GIMLET_CONTROLPLANE_BASE/v1/chat/completions?deviceID=$GIMLET_DEVICE_ID" \
+  -H "Authorization: Bearer $GIMLET_API_KEY" \
+  -H "Content-Type: application/json" \
+  --data '{
+    "model": "unknown",
+    "messages": [
+      {"role": "system", "content": ""},
+      {"role": "user", "content": "test-prompt"}
+    ],
+    "stream": true
+  }' \
+  -k >/dev/null 2>/dev/null
```

Comment:
has my controlplane base set wrong and the script just failed without printing anything, so might be fine to just remove the 2>/dev/null